zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -1053,8 +1053,8 @@ impl<'a, F: FieldExt> ConstraintBuilder<'a, F> {
             RwTableTag::TxLog,
             [
                 tx_id,
-                index + (1u64 << 8).expr() * log_id,
-                tag.expr(),
+                index + (1u64 << 32).expr() * field_tag.expr() + (1u64 << 48).expr() * log_id,
+                field_tag.expr(),
```

Comment:
why do we have duplicated `field_tag` in both fields?

Reply:
synced with spec, removed the second one . thanks !!